pkg/debugapi/balances.go

```diff
@@ -0,0 +1,77 @@
+// Copyright 2020 The Swarm Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package debugapi
+
+import (
+	"github.com/ethersphere/bee/pkg/jsonhttp"
+	"github.com/ethersphere/bee/pkg/swarm"
+	"github.com/gorilla/mux"
+	"sort"
+
+	"net/http"
+)
+
+type balanceResponse struct {
+	Peer    string `json:"peer"`
+	Balance int64  `json:"balance"`
+}
+
+type balancesResponse struct {
+	Balances []balanceResponse `json:"balances"`
+}
+
+func (s *server) balancesHandler(w http.ResponseWriter, r *http.Request) {
+
+	balances, err := s.Accounting.Balances()
+
+	if err != nil {
+		jsonhttp.InternalServerError(w, "Can not get balances")
+		s.Logger.Debugf("debug api: balances: %v", err)
+		s.Logger.Errorf("debug api: balances: %v", err)
+		return
+	}
+
+	var balResponses []balanceResponse
+
+	keys := make([]string, 0, len(balances))
```

Comment:
If the `keys` exist only to be sorted for the response, `keys` slice can be avoided (less allocations). `balResponses` can be appended with elements in one loop only and sorted with `sort.Slice` (or to implement `sort.Interface` and use `sort.Sort`) afterwards.